templates/component.yml.erb

```diff
@@ -0,0 +1 @@
+<%= @component.to_yaml %>
```

Comment:
If this is the whole template, why not use `stdlib::to_yaml($component)` in Puppet?

Reply:
Thanks, I've replaced that in code and dropped the templates at all